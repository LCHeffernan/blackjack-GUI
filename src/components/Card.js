import React from "react";

const Card = () => <div className="card-container">card</div>;

export default Card;
