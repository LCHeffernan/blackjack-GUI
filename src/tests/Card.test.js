import React, { render } from "@testing-library/react";
import Card from "../components/Card";

describe("Card", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Card />);
    expect(asFragment()).toMatchSnapshot();
  });
});
