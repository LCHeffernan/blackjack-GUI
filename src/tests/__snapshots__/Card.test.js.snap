// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<DocumentFragment>
  <div
    class="card-container"
  >
    card
  </div>
</DocumentFragment>
`;
